sklearn/metrics/pairwise.py

```diff
@@ -253,6 +253,7 @@ def euclidean_distances(
 
     Y : {array-like, sparse matrix} of shape (n_samples_Y, n_features), \
             default=None
+        If `None`, Y is set as a pointer to X.
```

Comment:
I probably wouldn't use the word pointer because it doesn't really have a meaning in Python. Maybe say "If `None` take `Y=X`" or something like that? Or "Uses Y=X"?

Reply:
Makes sense! I just changed it.